docs/sources/whatsnew/whats-new-in-v9-2.md

```diff
@@ -219,3 +219,17 @@ For more information, see the [SAML configuration documentation](https://grafana
 You can now map OAuth groups and roles to Server Admin for the GitLab, GitHub, AzureAD, Okta, and Generic OAuth integrations.
 To enable this functionality, set the `allow_assign_grafana_admin` configuration option to `true` in the desired OAuth integration section.
 For more information, see the [authentication configuration documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/) for each OAuth client.
+
+## Match parameter support in prometheus labels API
+
+Prometheus users running Prometheus 2.24 or higher can now elect to use the
```

Comment:
@gguillotte-grafana Can you take a look at this and let me know if you have any questions about how we can clean up this language?